src/db.ts

```diff
@@ -28,42 +51,76 @@ interface InternalReminder {
 	ping: string | null;
 }
 
-// -- Activity Tracker Database --
-export const activityTrackerBlacklistDb = new Enmap<string[]>("trackerBlacklist");
-activityTrackerBlacklistDb.ensure("general-user", []);
-activityTrackerBlacklistDb.ensure("general-game", []);
-
-// key: "[user id]-[game]"
-export const activityTrackerLogDb = new Enmap<ActivityLogEntry[], InternalActivityLogEntry[]>({
-	name: "trackerLog",
-	serializer: (data) => data.map((e) => ({ duration: e.duration.toISO(), date: e.date.toISO() })),
-	deserializer: (data) =>
-		data.map((e) => ({
-			duration: Duration.fromISO(e.duration),
-			date: DateTime.fromISO(e.date, { setZone: true }),
-		})),
-});
+/*
+-----------------------------
+- Activity Tracker Database -
+-----------------------------
+*/
 
-export interface ActivityLogEntry {
-	duration: Duration;
-	date: DateTime;
+/** Interface for tracker logs */
+export interface TrackerLog {
+	/** When a log got logged */
+	time: ISODate;
+	/** How long a game got played */
+	playtime: number;
+	/** User id of the User who just got logged */
+	userid: userID;
+	/** Game id of the game which just got logged */
+	gameName: string;
 }
 
-interface InternalActivityLogEntry {
-	duration: ISODuration;
-	date: ISODate;
+/** Interface for stripped down tracking logs */
+export interface TrackerSublog {
+	/** User- or Game- ID */
+	id: userID;
+	/** How often a User or Game got logged */
+	logs: number;
+	/** How long a User played a certain game
+	 * (Or the other way around)
+	 */
+	playtime: number;
 }
 
-// -- MISC --
-// Types
-export type ISODate = string; // ISO Date
-export type ISODuration = string; // ISO Duration
-export type jamID = string; // key of jamDb
-export type pollID = string; // key of pollDb
-export type proposalID = string; // key of proposalDb
-export type userID = string; // Discord user id
+/** Interface for tracking users */
+export interface TrackerUser {
+	/** How much a user played */
+	playtime: number;
```

Comment:
Should also be a DateTime.duration

Reply:
Same deal here, wanna leave as number -> mention in doc string that it is ms